change/@fluentui-react-components-2b36a74e-2147-4df0-a4f2-ab6c12c56033.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "Add slider package",
```

```suggestion
  "comment": "Export slider package",
```

or

```suggestion
  "comment": "Export slider package in /unstable",
```

---

I also don't see any added export in `react-components`, did I overlook?

---

oh, added the dep, but not the top level export. I'll fix that